data/wsl/create_unattend_iso.sh

```diff
@@ -0,0 +1,29 @@
+ #!/bin/bash
+
+if [ -z $1 ]; then
+    echo "You should supply the ISO file!"
+    echo "  USAGE: ${0} <ISO_FILE> <BIOS/UEFI>"
+    exit
+fi
+
+# if [$2 -ne "BIOS"]||[$2 -ne "UEFI"]; then
+#     echo "You should declare UEFI or BIOS!"
+#     echo "  USAGE: ${0} <ISO_FILE> <BIOS/UEFI>"
+#     exit
+# fi
+
+basename=`echo $1 | cut -d '.' -f 1`
+echo "***** Creating ${basename}_${2}_unattend.iso from ${1} *****"
```

Comment:
what should be the second parameter then?

Reply:
The second parameter is supposed to be 'UEFI' or 'BIOS' string. Anyway, I removed the bash script

Reply:
Pity, I found it useful and as good idea

Reply:
I'll add this script to our Confluence page, and maybe in the future we can create a job to auto generate the ISOs